test/controllers/trust.details.controller.spec.ts

```diff
@@ -457,6 +457,25 @@ describe('Trust Details controller', () => {
       expect(resp.text).toContain(TRUST_DETAILS_TEXTS.title);
       expect(resp.text).toContain(TRUST_DETAILS_TEXTS.subtitle);
       expect(resp.text).not.toContain(PAGE_TITLE_ERROR);
+      expect(resp.text).not.toContain(TRUST_NOT_ASSOCIATED_WITH_BENEFICIAL_OWNER_TEXT);
```

Comment:
Just wondering where the test is in this file to check that the field **is** there when flag is false?

Reply:
Strike that, I now see this is the registration test and it should never show on this journey.